x/incentives/types/expected_keepers.go

```diff
@@ -9,7 +9,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// BankKeeper defines the expected interface needed to retrieve account balances.
+// Defines the expected interface needed to retrieve account balances.
```

Comment:
As an FYI, the former is technically the golang standard. I don't think it or this comment actually matters here though

Reply:
Oh shoot I didn't realize this, so every comment describing a function in golang should start with the function name?

Reply:
My reasoning was this comment is showed when hovering over the function itself, so I felt like repeating the name was unnecessary. But if its the standard I will 100 percent add these back.

Reply:
Implemented golang comment standard on all function descriptions